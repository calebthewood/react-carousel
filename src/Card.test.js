import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

//Card({caption, src, currNum, totalNum})

/**
 * Tests that Card renders.
 */
it("renders without crashing", function () {
  // this is a low-value test, but better than nothing
  render(<Card
    caption="testCaption"
    src="testSrc"
    currNum="testNum"
    totalNum="totalNum" />);
});

/**
 * Snapshot Test:
 *
 */

it("matches snapshot", function () {
  const { container } = render(<Card
                                  caption="testCaption"
                                  src="testSrc"
                                  currNum="testNum"
                                  totalNum="totalNum"
                              />);
  expect(container).toMatchSnapshot();
});