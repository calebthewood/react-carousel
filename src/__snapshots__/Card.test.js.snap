// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testCaption
    </h4>
    <img
      alt="testCaption"
      class="Card-image"
      src="testSrc"
    />
    <small
      class="Card-small"
    >
      Image 
      testNum
       of 
      totalNum
      .
    </small>
  </div>
</div>
`;
